src/components/NewRuntimeModal.js

```diff
@@ -184,7 +184,7 @@ export const NewRuntimeModal = withModalDrawer({ width: 675 })(class NewRuntimeM
     return {
       cloudService: newRuntime.cloudService,
       ...(newRuntime.cloudService === cloudServices.GCE ? {
-        bootDiskSize: 50,
+        bootDiskSize: newRuntime.bootDiskSize,
```

Comment:
Is this already returned by leonardo? When i try to print this out locally it isn't defined in `newRuntime`, which has these fields:

May be worth providing a default value along the lines of `bootDiskSize: newRuntime.bootDiskSize ? newRuntime.bootDiskSize : someDefaultValue`

<img width="596" alt="Screen Shot 2021-04-16 at 2 22 09 PM" src="https://user-images.githubusercontent.com/6465084/115067634-33cdce80-9ebf-11eb-85e0-f7adf1a39a65.png">

Reply:
so I realize we don't actually display cost estimate when there's no runtime existent